### PR TITLE
I0593 miovision new cameras documentation

### LINKED ISSUE #593: Consolidate information on adding and removing cameras
There's this folder with a bare skeleton of a README https://github.com/CityofToronto/bdit_data-sources/tree/master/volumes/miovision/update_intersections

Meanwhile the below steps which were in the main Miovision README got archived

https://github.com/CityofToronto/bdit_data-sources/blob/23b97d3657fe5eda5318dc7984dce99fb06ede0a/volumes/miovision/Archive.md?plain=1#L31-L216

These should be consolidated in the `update_intersections` folder and that folder should be referenced from the main README

### PR DESCRIPTION
**Pull request by gabrielwol**
## What this pull request accomplishes:
- Consolidates information on adding/removing intersections under volumes/miovision/update_intersections/Readme.md
- Added some new sql snippets to help with process / QC. 
- Updated volumes/miovision/update_intersections/new_intersection_activation_dates.ipynb to use airflow connection for easier reuse. 

## Issue(s) this solves:
-Closes #593 

## What, in particular, needs to reviewed:
- How to proceed on updating these functions for backfilling: https://github.com/CityofToronto/bdit_data-sources/pull/668#discussion_r1289106744
- New Insert script for intersections: https://github.com/CityofToronto/bdit_data-sources/blob/4054a939c0db5ca9e4f5531b429e5f7db26e72c5/volumes/miovision/update_intersections/Readme.md?plain=1#L68-L100
- Suggested new criteria for selecting intersection_movements (volume::numeric / classification_volume >= 0.005) https://github.com/CityofToronto/bdit_data-sources/blob/4054a939c0db5ca9e4f5531b429e5f7db26e72c5/volumes/miovision/update_intersections/Readme.md?plain=1#L109-L143
- Suggested QC check for intersection_movements: https://github.com/CityofToronto/bdit_data-sources/blob/4054a939c0db5ca9e4f5531b429e5f7db26e72c5/volumes/miovision/update_intersections/Readme.md?plain=1#L176-L217
- Suggested QC check for volumes: https://github.com/CityofToronto/bdit_data-sources/blob/4054a939c0db5ca9e4f5531b429e5f7db26e72c5/volumes/miovision/update_intersections/Readme.md?plain=1#L263-L331

## What needs to be done by a sysadmin after this PR is merged
-Raph to give Gabe insert permissions in order to perform tasks associated with adding new cameras (#622, others to come)

### CODE REVIEW

File: volumes/miovision/update_intersections/Readme.md [hunk 1]
```diff
@@ -1,7 +1,401 @@
...
+	a) **Backfill `miovision_api.volumes`**   
+    If not already complete, use the [api script](../api/intersection_tmc.py) with `--pull` to backfill `miovision_api.volumes` table between the date_installed and previous day. 
+
+	b) **Backfill additional tables**  
+    We now have to run a couple of functions manually with (%s::date, %s::date) being (start_date::date, end_date::date) to finish aggregating the backfilled data.  
+	```sql
+	SELECT miovision_api.find_gaps(%s::date, %s::date);
+	SELECT miovision_api.aggregate_15_min_tmc(%s::date, %s::date);
+	SELECT miovision_api.aggregate_15_min(%s::date, %s::date); 
+	SELECT miovision_api.report_dates(%s::date, %s::date);
```
Comment (gabrielwol) [inline volumes/miovision/update_intersections/Readme.md:260]: Looking through #429 I realize some of these functions need updates to be used for backfilling. 
Either an option to run for only one intersection_uid like [Jedwin proposed in 2022](https://github.com/CityofToronto/bdit_data-sources/blob/fbfa27a51f500dade17840520e736b434f6066f6/volumes/miovision/sql/function-aggregate-volumes_15min_mvt_filter_65_66.sql): 
OR other updates so they don't insert duplicate data.
@radumas how would you recommend proceeding?

**Review by radumas — APPROVED** [review]
Looks good. I added some more info on intersection movements and fixed the formatting. Each section starting with a numbered list bugged me.

What markdown editor are you using? The automated TOC isn't correctly ignoring the title.

**Comment by radumas** [comment]
Also nice work with the text processing on intersection movements to make a nice output!

File: volumes/miovision/update_intersections/Readme.md [hunk 2]
```diff
@@ -1,7 +1,401 @@
...
-intersections.
+This readme contains information and resources on how to add/remove Miovision intersections from our data pipeline.  
+For the main Miovision readme, see [here](../README.md). 
 
-- `New Intersection Activation Dates.ipynb`: script to get first date of data
-for each new intersection.
\ No newline at end of file
+<!-- TOC -->
+
+- [Miovision Intersection Update Resources](#miovision-intersection-update-resources)
```
[the remark this answers is not in the document]
Comment (gabrielwol) [reply volumes/miovision/update_intersections/Readme.md:9]: I tried the top two results on VS Code marketplace for "markdown TOC" but neither seemed to work. Is there one you recommend?

[the remark this answers is not in the document]
Comment (radumas) [reply volumes/miovision/update_intersections/Readme.md:9]: I'm using https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

**Pull request merged**